From fa19b0d21f3e064adbdff907f06cd15b2cb81d16 Mon Sep 17 00:00:00 2001
From: Christian Mollekopf <mollekopf@kolabsys.com>
Date: Mon, 15 Jul 2013 09:32:21 +0000
Subject: Don't set the picture type.

Setting the type explicitly is not required and apparently breaks loading some images.

--- a/conversion/kabcconversion.cpp
+++ b/conversion/kabcconversion.cpp
@@ -365,7 +365,6 @@
     if (logo.isEmpty()) {
         Error() << "failed to read picture";
     }
-    logo.setType(fromStdString(mimetype));
     return logo;
 }
 
